docs/architecture.md

```diff
@@ -0,0 +1,93 @@
+# Project Architecture
+
+## Overview
+
+This document provides an overview of the architecture of our blockchain project. The project consists of a frontend built with Angular, and smart contracts written in Solidity and managed with Hardhat.
```

Comment:
I would suggest adding a simple explaination about the current project here as example when explaining the architecture so users will have idea on what we are building and why we need to build it using blockchain and not just a normal DB.

Reply:
Updated in https://github.com/OsmosysSoftware/supply-chain-blockchain/pull/5/commits/f0a9eea9645c3517951ce350477fd46392f1bff9